app/models/guest.rb

```diff
@@ -0,0 +1,3 @@
+class Guest < ApplicationRecord
+  has_many :rooms
```

Comment:
In order to add a `has_many :rooms` association here on the `Guest` model, that would mean that the `Room` model would have a `belongs_to :guest` association. But like we've learned, in order to have a `belongs_to :guest` association, it means that there is a `guest_id` foreign key on the rooms table, which there is not. 

You could have a `has_many :rooms` association here on the `Guest` model, but it would need to go through the join table to get there.